packages/api/cms-api/src/translation/translation.resolver.ts

```diff
@@ -0,0 +1,18 @@
+import { Inject } from "@nestjs/common";
+import { Args, Query, Resolver } from "@nestjs/graphql";
+
+import { TranslationConfig } from "./translation.config";
+import { TRANSLATION_CONFIG } from "./translation-config.constants";
+
+@Resolver(() => String)
+export class TranslationResolver {
+    constructor(@Inject(TRANSLATION_CONFIG) private readonly config: TranslationConfig) {}
+
+    @Query(() => String)
+    async translate(
```

Comment:
I'd change this to `contentTranslationTranslate(text: String!, targetLanguage: String!)`:

- `contentTranslation` as "module prefix"
- `text` we're translating text
- `targetLanguage`: `language` is too unspecific. Is it the source language or the target language?

Come to think of it, why do we even need this module in the library? It doesn't do anything than call the service provided by the application -> we could implement it in the application

Reply:
Removed the api module implementation with an example in the demo: 79838339a24b83de6a330fa352cf78e017fe226f